acs-engine.go

```diff
@@ -132,12 +132,25 @@ var artifactsDir = flag.String("artifacts", "", "directory where artifacts will
 var classicMode = flag.Bool("classicMode", false, "enable classic parameters and outputs")
 var parametersOnly = flag.Bool("parametersOnly", false, "only output the parameters")
 
+// AcsEngineBuildSHA is the Git SHA-1 of the last commit
+var AcsEngineBuildSHA string
```

Comment:
since we are adding commit hash, i highly suggest to add build time, too. It's a lot easier to see how dated the binary is.